utils/_context/_scenarios.py

```diff
@@ -645,13 +671,18 @@ def _wait_interface(self, interface, timeout):
         interface.wait(timeout)
 
     def _check_env_vars(self):
-        for env in ["DD_API_KEY", "DD_APP_KEY", "DD_API_KEY_2", "DD_APP_KEY_2", "DD_API_KEY_3", "DD_APP_KEY_3"]:
-            if env not in os.environ:
-                raise Exception(f"Please set {env}, OTel E2E test requires 3 API keys and 3 APP keys")
+        if self.include_intake:
+            assert all(
+                key in os.environ for key in ("DD_API_KEY_2", "DD_APP_KEY_2")
+            ), "OTel E2E test requires DD_API_KEY_2 and DD_APP_KEY_2"
+        if self.include_collector:
+            assert all(
+                key in os.environ for key in ("DD_API_KEY_3", "DD_APP_KEY_3")
+            ), "OTel E2E test requires DD_API_KEY_3 and DD_APP_KEY_3"
 
     @property
     def library(self):
-        return LibraryVersion("open_telemetry", "0.0.0")
+        return self.weblog_container.library
```

Comment:
So, if I understand correctly, `open_telemetry` was hardcoded, the real library identifier is now `java_otel` for both old `OTEL_TRACING_E2E` ...  and the new `OTEL_INTEGRATIONS` scenarios?

Reply:
Yes, I have also changed the value for the @missing_feature for otel_tracing_e2e tests.
We have only the open_telemetry tests for Java, but In the future we should have tests for other languages....